### ISSUE TITLE
Add a Travis CI badge to the Inventories README

### ISSUE THREAD

**Issue by cslecrone** [nyu-devops-summer-2021-inventory/inventories#39]
**As a** developer
**I need** a build and CodeCov badges in my repository's README
**So that** I know that `main` is currently working and covered by unit tests

**Assumptions**
* We'll need badges for both Travis CI and CodeCov

**Acceptance criteria**
* Our GitHub repo will have prominent build and test badges that accurately reflect the state of our build and tests.

**Comment by cslecrone** [comment]
I'm waiting on #67 to land before putting in another PR to address this.
